includes/admin/class-sensei-learners-main.php

```diff
@@ -1221,20 +1221,17 @@ public function add_learners_box() {
 			<div class="inside">
 				<form name="add_learner" action="" method="post">
 					<p>
-						<select name="add_user_id[]" id="add_learner_search" multiple="multiple" style="min-width:300px;">
-						</select>
+						<select name="add_user_id[]" id="add_learner_search" multiple="multiple" style="min-width:300px;"></select>
 						<?php if ( 'lesson' === $form_post_type ) { ?>
-							<label for="add_complete_lesson"><input type="checkbox" id="add_complete_lesson" name="add_complete_lesson"  value="yes" /> <?php esc_html_e( 'Complete lesson for student', 'sensei-lms' ); ?></label>
-						<?php } elseif ( 'course' === $form_post_type ) { ?>
-							<label for="add_complete_course"><input type="checkbox" id="add_complete_course" name="add_complete_course"  value="yes" /> <?php esc_html_e( 'Complete course for student', 'sensei-lms' ); ?></label>
-						<?php } ?>
-						<br/>
-						<span class="description"><?php esc_html_e( 'Search for a user by typing their name or username.', 'sensei-lms' ); ?></span>
+							<div><label for="add_complete_lesson"><input type="checkbox" id="add_complete_lesson" name="add_complete_lesson"  value="yes" /> <?php esc_html_e( 'Complete lesson for selected student(s)', 'sensei-lms' ); ?></label></div>
```

Comment:
Are the extra `div`s truly needed here? It seemed to work fine when I removed them.

Reply:
In the design, the checkbox and the label are below the search box, so here I wanted to have the checkbox and the label in a block to move them below instead of beside the box as they are now. We can actually achieve this with CSS, then we won't need the div to make it look like the new design 👍